components/logins/src/engine.rs

```diff
@@ -204,27 +202,23 @@ mod test {
         let a_id = engine.add(a.clone()).expect("added a");
         let b_id = engine.add(b.clone()).expect("added b");
 
-        assert_eq!(a_id, a.guid);
-
-        assert_ne!(b_id, b.guid, "Should generate guid when none provided");
-
+         //Get item a's ID, make sure no errors getting a and for a's ID to exist - ben and ryan
         let a_from_db = engine
             .get(&a_id)
             .expect("Not to error getting a")
             .expect("a to exist");
 
+
         assert_logins_equiv(&a, &a_from_db);
-        assert_ge!(a_from_db.time_created, start_us);
+        assert_ge!(a_from_db.time_created, start_us); //make sure our times created/password changed/last used are correct - ben  and ryan
```

Comment:
Some of these comments are useful, but they should be more descriptive/more precisely written.

In general, it's nice to have a comment at the beginning of the test outlining what it tests, and then have comments before logical blocks of code saying what they are doing.

I agree that this could use comments, maybe take that approach.